static/js/components/UserPageAboutMeDialog_test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { assert } from 'chai';
+import sinon from 'sinon';
+import _ from 'lodash';
+
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import TestUtils from 'react-addons-test-utils';
+
+import UserPageAboutMeDialog from './UserPageAboutMeDialog';
+import { USER_PROFILE_RESPONSE } from '../constants';
+
+describe('UserPageAboutMeDialog', () => {
+  let sandbox;
+  let defaultRowProps, setUserPageDialogVisibility, clearProfileEdit, saveProfile;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    setUserPageDialogVisibility = sandbox.stub();
+    clearProfileEdit = sandbox.stub();
+    saveProfile = sandbox.stub();
+    saveProfile.returns({
+      then: () => {}
+    });
+
+    defaultRowProps = {
+      profile: USER_PROFILE_RESPONSE,
+      setUserPageAboutMeDialogVisibility: setUserPageDialogVisibility,
+      clearProfileEdit: clearProfileEdit,
+      saveProfile: saveProfile,
+      ui: {
+        userPageAboutMeDialogVisibility: true
+      }
+    };
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  const getDialog = () => document.querySelector('.about-me-dialog');
+  const renderDialog = () => (
+    mount (
+      <MuiThemeProvider muiTheme={getMuiTheme()}>
+        <UserPageAboutMeDialog {...defaultRowProps} />
+      </MuiThemeProvider>,
+      {
+        context: { router: {}},
+        childContextTypes: { router: React.PropTypes.object }
+      }
+    )
+  );
+
+  it('render dialog with data', () => {
+    defaultRowProps['profile'] = Object.assign(_.cloneDeep(USER_PROFILE_RESPONSE), {
+      about_me: "Hello world"
+    });
+    renderDialog();
+    assert.equal(document.querySelector('h3.dialog-title').textContent, "About Me");
+    assert.equal(document.querySelector("textarea").textContent, "Hello world");
+  });
+
+  it('render dialog without data', () => {
+    renderDialog();
+    assert.equal(document.querySelector('h3.dialog-title').textContent, "About Me");
+    assert.equal(document.querySelector("textarea").textContent, "");
+  });
+
+  it('render dialog when visibility det to false', () => {
```

Comment:
Can we add a test that, when viewing a profile that is _not_ your own, when the "about me" value is unset, the section doesn't show up at all? Or is that what this test is doing? I'm not clear on what this `userPageAboutMeDialogVisibility` property is actually for.